oe_theme.theme

```diff
@@ -1219,6 +1219,9 @@ function oe_theme_theme_suggestions_form_alter(array &$suggestions, array $varia
  * Implements hook_preprocess_input().
  */
 function oe_theme_preprocess_input__search(array &$variables): void {
+  if ($variables['element']['#id'] === 'admin-toolbar-search-input') {
```

Comment:
As I understand we don't need that if used approach suggested https://github.com/openeuropa/oe_theme/issues/1176#issuecomment-1268392745